source/fe/mapping_c1.cc

```diff
@@ -66,7 +66,7 @@ MappingC1<2>::MappingC1Generic::add_line_support_points (const Triangulation<2>:
                                                          std::vector<Point<2> > &a) const
 {
   const unsigned int dim = 2;
-  std::vector<Point<dim> > line_points (2);
+  static const QGaussLobatto<1> quad_points(4);
```

Comment:
Since the use of Gauss-Lobatto points is hardcoded here we may as well write it out instead of using the static object. I'll fix this.